ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -806,7 +810,15 @@
         ;; Route listing with number of changes
         "Taulukossa on listattu valitussa palvelussa havaittuja muutoksia. Voit valita listalta yhden reitin kerrallaan tarkasteluun. Valitun reitin reitti- ja aikataulutiedot näytetään taulukon alapuolella kalenterissa, kartalla, vuorolistalla ja pysäkkiaikataululistalla."
 
-        [route-changes e! (:gtfs/route-changes changes) selected-route route-hash-id-type]]]
+        [:div.row
+         [:div.col-md-6 ""]
+         [:div.col-md-6 {:style {:margin-top "10px"}}
```

Comment:
`rem` should be used by default for all layout measurements, px only if there are problems with rem/em.

Reply:
Changed to use flex